cpp/src/Slice/Parser.cpp

```diff
@@ -1193,11 +1193,24 @@ Slice::Container::lookupException(const string& scoped, bool printError)
         ExceptionPtr ex = ExceptionPtr::dynamicCast(p);
         if (!ex)
```

Comment:
This code does not make sense. We should not allow type aliases for exceptions at all.

Reply:
True, I got carried away while adding the code for classes and structs. I'll remove this.

Reply:
This was removed.